source/_integrations/wiz.markdown

```diff
@@ -93,6 +93,27 @@ A power monitoring sensor is available on devices with the following hardware mo
 - ESP20_SHRGB_31R
 - ESP20_SHTW_31R
 
+Please note: the power monitoring sensor reports in Watts (W), rather than kiloWatts per hour (kWh). As a result, it is not compatible with the Energy Dashboard. There is currently a workaround for this:
+
+1. Go to Settings -> Devices & Services and click on the + button to add a new integration.
```

Comment:
```suggestion
1. Go to {% my integrations title="**Settings** > **Devices & Services**" %} and click on the + button to add a new integration.
```
We have these **My** links that take you directly to this section in your Home Assistance UI :star_struck: 
And we use bold to markup UI elements.